sklearn/neighbors/lmnn.py

```diff
@@ -263,6 +270,10 @@ def fit(self, X, y):
         # Compute the gradient part contributed by the target neighbors
         grad_static = self._compute_grad_static(X_valid, target_neighbors)
 
+        # Compute the regularization coefficient
```

Comment:
Again, calling this normalized mu the regularization coefficient here may be also a bit confusing

Reply:
I wrote it now as pull loss coefficient.